grails-app/views/report/showRequestDetailReport.gsp

```diff
@@ -0,0 +1,284 @@
+<script type="text/javascript" charset="utf8" src="//cdnjs.cloudflare.com/ajax/libs/datatables/1.9.4/jquery.dataTables.js"></script>
+<script>
+    function initializeDataTable() {
+      var options = {
+        "bDestroy": true,
+        "bProcessing": true,
+        "iDisplayLength": 100,
+        "bSearch": false,
+        "bScrollCollapse": true,
+        "bJQueryUI": true,
+        "bAutoWidth": true,
+        "bScrollInfinite": true,
+        "sScrollY": 500,
+        "sPaginationType": "two_button",
+        "sAjaxSource": "${request.contextPath}/json/getRequestDetailReport",
+        "fnServerParams": function ( data ) {
+          data.push({ name: "destination.id", value: $("#destinationId").val() });
+          data.push({ name: "origin.id", value: $("#originId").val() });
+          data.push({ name: "dateIssuedFrom", value: $("#dateIssuedFrom").val() });
+          data.push({ name: "dateIssuedTo", value: $("#dateIssuedTo").val() });
+          data.push({ name: "category", value: $("#category").val() });
+          data.push({ name: "tags", value: $("#tags").val() });
+          data.push({ name: "catalogs", value: $("#catalogs").val() });
+          data.push({ name: "product", value: $("#product").val() });
+          data.push({ name: "reasonCode", value: $("#reasonCode").val() });
+        },
+        "fnServerData": function ( sSource, aoData, fnCallback ) {
+          $.ajax( {
+            "dataType": 'json',
+            "type": "POST",
+            "url": sSource,
+            "data": aoData,
+            "success": fnCallback,
+            "timeout": 30000,
+            "error": handleAjaxError,
+            beforeSend : function(){
+              $(".loading").show();
+            },
+            complete: function(){
+              $(".loading").hide();
+            },
+          } );
+        },
+        "oLanguage": {
+          "sZeroRecords": "No records found",
+          "sProcessing": "Loading <img alt='spinner' src='${request.contextPath}/images/spinner.gif' /> Loading... "
+        },
+        "aLengthMenu": [
+          [5, 15, 25, 100, 1000, -1],
+          [5, 15, 25, 100, 1000, "All"]
+        ],
+        "aoColumns":
+          [
+            { "mData": "requestNumber" },
+            { "mData": "dateRequested" },
+            { "mData": "dateIssued" },
+            { "mData": "origin" },
+            { "mData": "destination" },
+            { "mData": "productCode" },
+            { "mData": "productName" },
+            { "mData": "qtyRequested"},
+            { "mData": "qtyIssued"},
+            { "mData": "reasonCodes"},
+            { "mData": "qtyDemand"},
+          ],
+        "dom": '<"top"i>rt<"bottom"flp><"clear">',
+        "aaSorting": [[ 0, "asc" ]],
+      };
+
+      $('#requestDetailReportTable').dataTable(options);
+    }
+
+    function handleAjaxError( xhr, status, error ) {
+      if ( status === 'timeout' ) {
+        alert( 'The server took too long to send the data.' );
```

Comment:
How about using $.notify("The server took too long to send the data.'", "error");

Reply:
+1